### ISSUE TITLE
Create a figure to accompany the ITS analysis

### ISSUE THREAD

**Issue by mbcann01** [brad-cannell/detect_pilot_test_1y#14]

**Comment by mbcann01** [comment]
My first attempt at this is in a32b7affbb7c3f066fc4f53ed74086dcb04915e1

The plot still needs some work. It's working now with gridExtra, but it might like better with faceting.

**Comment by mbcann01** [comment]
* Originally, I didn't like the facets because it cut off part of the annotations for the shaded areas
* I tried moving the annotations down, but then they overlapped the lines
* I tried changing the y-axis limits, but couldn't figure out a straightforward way to do that for each individual facet.
* I tried making 4 individual plots and combining them with gridExtra, but I wasn't crazy about the way that looked. Specifically, the way the axis titles look.
* Now I'm thinking I should go back to using facets and just use something other than annotations (subtitle) to identify the shaded areas on the plot.

- [ ] Try out facets with no annotations on the shaded areas
- [ ] Try out scatter plot
- [ ] Try out mean lines
